crates/encoding/src/encoding.rs

```diff
@@ -306,13 +306,15 @@ impl Encoding {
         if gradient.p0 == gradient.p1 && (gradient.r0 - gradient.r1).abs() < SKIA_EPSILON {
             self.encode_color(DrawColor::new(Color::TRANSPARENT));
```

Comment:
I'd be tempted to just make this condition return

Either as `return self.encode_color`, or by adding a `return;` statement.
That matches the semantics of this being an early-exit condition

Reply:
Sounds good, added here I believe (and fixed formatting)